server/src/client_api.rs

```diff
@@ -507,6 +514,52 @@ mod tests {
         }
     }
 
+    #[tokio::test]
+    async fn response_includes_variant_stickiness_for_strategy_variants() {
+        let features_cache: Arc<DashMap<String, ClientFeatures>> = Arc::new(DashMap::default());
+        let token_cache: Arc<DashMap<String, EdgeToken>> = Arc::new(DashMap::default());
+        let app = test::init_service(
+            App::new()
+                .app_data(Data::from(features_cache.clone()))
+                .app_data(Data::from(token_cache.clone()))
+                .service(web::scope("/api/client").service(get_features)),
+        )
+        .await;
+
+        features_cache.insert("production".into(), cached_client_features());
+        let mut production_token = EdgeToken::try_from(
+            "*:production.03fa5f506428fe80ed5640c351c7232e38940814d2923b08f5c05fa7".to_string(),
```

Comment:
I think I added some test helper methods here for creating client tokens already validated, also, you don't need to use an actual secret. Anything behind the . could probably just be 'somesecret'.